sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/api/Azure.ResourceManager.CognitiveServices.netstandard2.0.cs

```diff
@@ -373,12 +406,21 @@ public partial class AvailableCognitiveServicesSku
         public System.Collections.Generic.IReadOnlyList<Azure.ResourceManager.CognitiveServices.Models.CognitiveServicesSkuRestrictions> Restrictions { get { throw null; } }
         public string Tier { get { throw null; } }
     }
+    public partial class CapacityConfig
```

Comment:
`CapacityConfig` is too generic, consider renaming it to `CongnitiveServicesCapacityConfig`.